test/Sentry.Unity.Tests/IntegrationTests.cs

```diff
@@ -98,6 +97,8 @@ public IEnumerator BugFarmScene_EventCaptured_IncludesApplicationVersionAsReleas
 
             // assert
             Assert.AreEqual(Application.version, testEventCapture.Events.First().Release);
+
+            // PlayerSettings.productName = originalProductName;
```

Comment:
See: https://github.com/getsentry/sentry-unity/pull/230#pullrequestreview-686412931

```suggestion
            // PlayerSettings.productName = originalProductName;
        }

        [UnityTest]
        public IEnumerator BugFarmScene_EventCaptured_UserNameIsMachineNameWithDefaultPii()
        {
            yield return SetupSceneCoroutine("1_BugFarmScene");
        
            var testEventCapture = new TestEventCapture();
            using var _ = InitSentrySdk(o =>
            {
                o.AddIntegration(new UnityApplicationLoggingIntegration(eventCapture: testEventCapture));
                o.SendDefaultPii = true;
                o.IsEnvironmentUser = true;
            });
            var testBehaviour = new GameObject("TestHolder").AddComponent<TestMonoBehaviour>();
        
            testBehaviour.gameObject.SendMessage(nameof(testBehaviour.TestException));
        
            // assert
            Assert.Equals(Environment.UserName, testEventCapture.Events.First().User.Username);
```